go/buildhelper/lib.go

```diff
@@ -0,0 +1,43 @@
+package buildhelper
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+var CONNECTION_TIMEOUT = 15 * time.Second
+
+// EthAPI struct
+// based on https://eth.wiki/json-rpc/API
```

Comment:
@otherview I didn't understand this comment. In what way is it based on the contents of that page?

Reply:
This might be cleaned up. Ideally, we want a lib that wraps the json-rpc API of ethereum - this was a weak attempt of writing it down 🙃

Reply:
Outdated :)